src/components/App.js

```diff
@@ -8,8 +8,8 @@ import "../css/app.css";
 import initializeDeck from "../helpers/initializeDeck";
 import shuffleDeck from "../helpers/shuffleDeck";
 
-function App() {
-  //useState returns current state and async function to update state
+const App = () => {
+  //useState returns current state and function to update state
```

Comment:
If you'd like, you can add the link to the documentation on this:

https://reactjs.org/docs/hooks-state.html#declaring-a-state-variable